### PR TITLE
Documented supported format capabilities

### LINKED ISSUE #2148: Document capabilities supported for each file format
Not every supported file format has the same capabilities. It would be handy if the documentation indicates what features each format supports. Something like this:

| Format | Linguality | Comments | Context | Location |
| ---- | --- | --- | --- | --- |
| gettext | bi | yes | yes | yes |
| monolingual gettext | mono | yes | yes | yes |
| JSON | mono | no | no | no |
| CSV | bi | no | no | no |
| YAML | mono | ? | no | no |

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/61401746-document-capabilities-supported-for-each-file-format?utm_campaign=plugin&utm_content=tracker%2F253393&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F253393&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Naruto0**
Fixes #2148

Signed-off-by: Filip Hron <filip.hron@gmail.com>

### CODE REVIEW

File: docs/formats.rst [hunk 1]
```diff
@@ -44,6 +44,51 @@ Weblate can automatically detect several widely spread file formats, but this
...
+Below are listed capabilities of all supported formats.
+
++---------------------+------------+----------+---------+----------+
+| Format              | Linguality | Comments | Context | Location |
++=====================+============+==========+=========+==========+
+| :ref:`gettext`      | bilingual  | yes      | yes     | yes      |
++---------------------+------------+----------+---------+----------+
+| :ref:`mono_gettext` | mono       | yes      | yes     | yes      |
++---------------------+------------+----------+---------+----------+
+| :ref:`xliff:        | both       | yes      | yes     |          |
```
Comment (nijel) [inline docs/formats.rst]: Typo here, see [failed build on travis](https://travis-ci.org/WeblateOrg/weblate/jobs/425769749):

```
weblate/docs/formats.rst:62:Inline interpreted text or phrase reference start-string without end-string.
```

Comment (Naruto0) [reply docs/formats.rst]: Thank you!

Fixed.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Comment by wichert** [comment]
@Naruto0 Thanks! That is exactly what I was looking for.

**Comment by Naruto0** [comment]
@nijel You're welcome
@wichert Glad i could help

**Pull request merged**